atdd/src/lib/Codegen.ml

```diff
@@ -510,7 +510,7 @@ auto toJsonString(T)(T obj)
   |}
     atd_filename
     atd_filename
-    (Filename.remove_extension atd_filename)
+    (sprintf "%s_atd" (Filename.remove_extension atd_filename))
```

Comment:
I'm guessing this is a breaking change, but given that atdd wasn't officially announced it shouldn't be a real problem

Reply:
oops, I started release 2.13.0 without merging this PR. I'm planning to make another release in a few days.

Reply:
I added a changelog entry for this PR. Please fix it as needed. https://github.com/ahrefs/atd/pull/355/commits/2f4418c19c0f2e5460cb756cdfc072c609f59efe